src/components/Book/Book.tsx

```diff
@@ -0,0 +1,19 @@
+import { useState } from "react";
+import "./Book.scss";
+import { BookType } from "../BookList/BookList";
+
+export const Book = (props: BookType) => {
+  const [bookImage, setBookImage] = useState("./book-cover-placeholder.png");
+
+  return (
+    <>
```

Comment:
you have one parent name div with class book - so this react fragment is unnecessary